js/tasks.js

```diff
@@ -0,0 +1,11 @@
+import { appearBtn, closeBtn, hideBtn } from "./modules/login.js" 
+
+window.appearBtn = function() {
+    appearBtn()
+}
+window.closeBtn = function() {
+    closeBtn()
+}
+window.hideBtn = function() {
+    hideBtn()
+}
```

Comment:
update used functions to use parameters, and use them sending the corresponding parameters to the functions